README.md

```diff
@@ -0,0 +1,45 @@
+# Source for [rust-for-rustaceans.com](https://rust-for-rustaceans.com)
+
+Home of Reviews and Errata for ["Rust for Rustaceans" by Jon Gjengset](https://nostarch.com/rust-rustaceans).
+
+<kbd><img src="https://github.com/jonhoo/rust-for-rustaceans.com/blob/main/RustforRustaceans_cover.png?raw=true" title="Rust for Rustaceans" alt="book cover art with excited Ferris (Rust mascot)" /></kbd>
```

Comment:
```suggestion
![book cover art with excited Ferris (Rust mascot)](RustforRustaceans_cover.png?raw=true "Rust for Rustaceans")
```
I think this should work

Reply:
It looks much better with the border. But, as you wish.